internal/controller/driver_controller.go

```diff
@@ -487,6 +549,7 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 								Name:            fmt.Sprintf("csi-%splugin", r.driverType),
 								Image:           r.images["plugin"],
 								ImagePullPolicy: imagePullPolicy,
+								SecurityContext: securityContext,
```

Comment:
we also need to drop all the capability as well

```
Capabilities: &corev1.Capabilities{
	Drop: []corev1.Capability{"All"},
	},
```

Reply:
@Madhu-1 with that I remember we have a separate setting for running `provisioner` in privileged,
With the current change I am just looking to if the `logrotate is enabled`

So better would I should check if the logrotate is enabled as well as the spec setting is set we should enable securityContext right?

Reply:
if we are setting the securityContext we need to do the above as well to keep the minimal one. by default it wont drop all the capabilities.

Reply:
Done